### DISCUSSION TITLE
Keine Verbindung zur RaspberryMatic nach Internet-Trennung / Neustart FritzBox

### DISCUSSION THREAD

**Discussion by Dozer-hh** [OpenCCU/OpenCCU#1225] [🇩🇪 Fragen&Antworten]
Seit dem letzten Update auf RaspberryMatic 3.57.4.20210320 habe ich folgendes Problem:
Wenn die Fritzbox die Internetverbindung verliert, z.B. durch Firmware-Update / Verbindungsprobleme / Neustart verliert die HomeMatic Zentrale die Verbindung und baut diese auch nach Wiederherstellung der Internetverbindung / Neustart der FritzBox nicht wieder auf.

Somit ist kein Zugriff auf die Zentrale mehr möglich, weder von außerhalb, noch lokal.
Erst ein kompletter Neustart des Raspberry führt zur erneuten Verbindung.

Mein System:
RaspberryMatic 3.57.4.20210320 auf Raspberry Pi 3 B+
Fritzbox 7490 mit FRITZ!OS 07.24-87448
(Vorher: FRITZ!OS 07.24-87308)

**Comment by friedpa** [comment]
Das ist ein Diskussionspunkt für das Forum (Nochmal die selbe Frage stellen ändert nichts)
Ich kann dieses Verhalten NICHT bestätigen. Probiert mit meinem Testsystem RPI 3B+ und meinem Produktivsystem RPI 4
Fritzbox 6890 mit Fritz!OS 07.24-86513

**Reply by Dozer-hh** [reply]
> Das ist ein Diskussionspunkt für das Forum (Nochmal die selbe Frage stellen ändert nichts)

Nochmal die selbe Frage aufgrund der folgenden Anleitung: 

> Tut mir leid, aber das hier ist kein Supportforum, sondern ein Bug/Issuetracker, d.h. für Dinge die DEFINITIV ein Bug darstellen oder Funktionserweiterungen sind. Daher bitte entweder hier im Diskussusionsteil des GitHub Projektes einen neuen Beitrag eröffnen oder das externe RaspberryMatic Forum (https://homematic-forum.de/) dafür nutzen. Dies stand übrigens auch so im Ticket Template!

Daher also "hier im Diskussusionsteil des GitHub Projektes"!
Nach meiner Definition wäre das auch ein "Issue" / Bug, aber darüber lohnt sich die Diskussion nicht, da ich die eh verliere.


> Ich kann dieses Verhalten NICHT bestätigen. Probiert mit meinem Testsystem RPI 3B+ und meinem Produktivsystem RPI 4
> Fritzbox 6890 mit Fritz!OS 07.24-86513

Danke fürs Testen!
